src/Builder/FormContractor.php

```diff
@@ -94,9 +94,14 @@ public function getDefaultOptions($type, FieldDescriptionInterface $fieldDescrip
         if ($this->checkFormType($type, [
             'sonata_type_model',
             'sonata_type_model_list',
+            'sonata_type_model_hidden',
+            'sonata_type_model_autocomplete',
         ]) || $this->checkFormClass($type, [
             'Sonata\AdminBundle\Form\Type\ModelType',
+            'Sonata\AdminBundle\Form\Type\ModelTypeList',
```

Comment:
please don't add this class, it is deprecated.
If you are using this class, use `Sonata\AdminBundle\Form\Type\ModelListType` instead